src/tests/ftest/util/soak_utils.py

```diff
@@ -590,7 +589,7 @@ def launch_exclude_reintegrate(self, pool, name, results, args):
             tgt_idx = None
         else:
             target_list = random.sample(range(0, 8), targets)
-            tgt_idx = "{}".format(','.join(str(tgt) for tgt in target_list))
+            tgt_idx = f"{','.join(str(tgt) for tgt in target_list)}"
```

Comment:
`str.join` returns a `str`, so the f-string isn't necessary
```suggestion
            tgt_idx = ','.join(str(tgt) for tgt in target_list)
```
Or, if you want to use Phil's new-ish function
```suggestion
            from general_utils import list_to_str
            tgt_idx = list_to_str(target_list)
```